index.js

```diff
@@ -116,13 +116,12 @@ KafkaLogger.prototype.log = function(level, msg, meta, callback) {
         }
     }
     logMessage.level = level;
-    try {
-        logMessage.msg = msg + ' ' + JSON.stringify(meta);
-    } catch (e) {
-        logMessage.msg = msg + ' bad meta object of type ' +
-                (typeof (meta) === 'object' ? meta.constructor.name :
-                typeof (meta)) + ' ' + e.message;
+    var jsonMsg = {}
+    jsonMsg.text = msg;
```

Comment:
we might want to make this more unique `_text` ? I've started a convention of adding `_pid` and `_hostname` to messages to avoid conflicts (i.e. prefix with _ )